packages/git/src/browser/diff/git-diff-frontend-module.ts

```diff
@@ -30,5 +31,6 @@ export function bindGitDiffModule(bind: interfaces.Bind): void {
     }));
 
     bindViewContribution(bind, GitDiffContribution);
+    bind(TabBarToolbarContribution).toService(GitDiffContribution);
```

Comment:
`inSingletonScope`

Reply:
I verified the source code, each example binding for `TabBarToolBarContribution` does not set the `inSingletonScope`. Is this correct?

For example:
https://github.com/theia-ide/theia/blob/566ebda6b9ede33984ff0d06044e889b2191c9d7/packages/debug/src/browser/debug-frontend-module.ts#L91

https://github.com/theia-ide/theia/blob/566ebda6b9ede33984ff0d06044e889b2191c9d7/packages/keymaps/src/browser/keymaps-frontend-module.ts#L39

https://github.com/theia-ide/theia/blob/566ebda6b9ede33984ff0d06044e889b2191c9d7/packages/navigator/src/browser/navigator-frontend-module.ts#L45

Reply:
ah, it is `toService`, not `to`, yes then it is correct